src/Lykke.Service.NeoSign/Helpers/Transaction/TransactionSerializer.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Common;
+
+namespace Lykke.Service.NeoApi.Helpers.Transaction
+{
+    public static class TransactionSerializer
+    {
+        public static string Serialize(NeoModules.NEP6.Transactions.Transaction transaction)
+        {
+            return TransactionContract.FromDomain(transaction).ToJson();
```

Comment:
Let's encode json to the base64 string. This simplifies manual requests execution.